includes/admin/payments/view-payment-details.php

```diff
@@ -384,7 +384,7 @@
 															'name'             => 'give-variable-price',
 															'chosen'           => true,
 															'show_option_all'  => '',
-															'show_option_none' => '',
+															'show_option_none' => ( '' === get_post_meta( $payment_id, '_give_payment_price_id', true ) ? __( 'None', 'give' )  : '' ),
```

Comment:
@DevinWalker what will be the donation level dropdown default value when donation level does not match custom and price level. Currently, I set it to an empty string and it will appear as `None` in donation level dropdown.